src/driver.rs

```diff
@@ -188,12 +190,31 @@ pub fn main() {
 
     exit(rustc_driver::catch_with_exit_code(move || {
         let mut orig_args: Vec<String> = env::args().collect();
-        let has_sysroot_arg = arg_value(&orig_args, "--sysroot", |_| true).is_some();
+
+        let has_sysroot_arg = |args: &mut Vec<String>| {
+            if arg_value(&args, "--sysroot", |_| true).is_some() {
+                return true;
+            }
+            // https://doc.rust-lang.org/rustc/command-line-arguments.html#path-load-command-line-flags-from-a-path
+            // Beside checking for existence of `--sysroot` from the args on the command line, we need to
+            // check for the arg files that are prefixed with @ as well to be consistent with rustc
+            for arg in args.iter() {
+                if let Some(arg_file_path) = arg.strip_prefix("@") {
+                    if let Ok(arg_file) = read_to_string(arg_file_path) {
+                        let split_arg_file: Vec<String> = arg_file.split("\n").map(|s| s.to_string()).collect();
```

Comment:
Please use `lines()` instead of `spilt` here, as according to the docs, windows line endings are also allowed in arg files

```suggestion
                        let split_arg_file: Vec<String> = arg_file.lines().map(|s| s.to_string()).collect();
```

I like the usage of `arg_value` also for this!

Reply:
Done.